homeassistant/components/sensor/__init__.py

```diff
@@ -17,13 +17,15 @@
 
 ENTITY_ID_FORMAT = DOMAIN + '.{}'
 
+GROUP_NAME_ALL_SENSORS = 'all sensors'
```

Comment:
Problem with this one is that every time an entity of this automatica group changes state, the group will also change state. So effectively we would have every state change twice in the state machine.

Reply:
hm. interesting. Probably also the reason this one hasn't been added up until now.
I don't understand the core of HA quite yet so... any suggestions on how to get that done properly?